noxfile.py

```diff
@@ -145,7 +171,18 @@ def test_examples(session: nox.Session) -> None:
         )
 
 
-@nox.session(python=["3.10", "3.11"], tags=["next"])
+@nox_utils.customize_session(
+    python=PYTHON_VERSIONS,
+    tags=["next"],
+    env_vars=["NUM_PROCESSES"],
+    ignore_paths=[  # Skip when only gt4py.cartesian or doc files have been updated
```

Comment:
I guess you are experimenting with ignore_paths vs paths? Or is there any reason why you use ignore_paths for next and paths for cartesian?

Reply:
This is mimicking what we currently have on CI: `paths-ignore` for _cartesian_ and _next_, and `paths` for _eve_ and _storage_